irmaclient/client.go

```diff
@@ -559,6 +565,9 @@ func (client *Client) rawLogEntriesToLogInfo(entries []*LogEntry) ([]LogInfo, er
 
 func (client *Client) SetPreferences(prefs Preferences) {
 	client.irmaClient.SetPreferences(prefs)
+	if prefs.DeveloperMode {
+		client.openid4vpClient.eudiConf.EnableStagingTrustAnchors()
```

Comment:
Maybe an idea to immediately update the revocation lists when activating staging anchors?
Otherwise, it might take up to half an hour for the lists to update themselfs.